autorag/data/parse/base.py

```diff
@@ -32,10 +35,19 @@ def wrapper(
 				result = func(
 					data_path_list=data_path_list, parse_method=parse_method, **kwargs
 				)
-		elif func.__name__ in ["clova_ocr", "llama_parse"]:
+		elif func.__name__ in ["clova_ocr", "llama_parse", "table_hybrid_parse"]:
 			result = func(data_path_list=data_path_list, **kwargs)
 		else:
 			raise ValueError(f"Unsupported module_type: {func.__name__}")
+		result = _add_date(result)
 		return result
 
 	return wrapper
+
+
+def _add_date(result):
+	date_lst = [datetime.now()] * len(result[0])
+	result_lst = list(result)
+	result_lst.append(date_lst)
+	result_tuple = tuple(result_lst)
+	return result_tuple
```

Comment:
I think you need to get a last modified datetime from each file.
There is `get_file_metadata` function at `autorag/data/utils/util.py`
Check that out